types/sizzle/index.d.ts

```diff
@@ -3,6 +3,15 @@ export as namespace Sizzle;
 declare const Sizzle: SizzleStatic;
 export = Sizzle;
 
+// For users who don't have "dom" types
+/* eslint-disable @typescript-eslint/no-empty-interface */
+declare global {
+    interface Document {}
```

Comment:
Is there no better way? Like, is there an interface that these satisfy that the library should accept instead?

Reply:
I don't know of a good way to solve for that 😬. In theory I think the library only guarantees these work with the actual DOM types: either from browsers or a vdom that implements the spec. Maybe [`@types/web`](https://www.npmjs.com/package/@types/web) is the answer here? But I don't know about how reasonable it is to take that on as a (dev) dependency.

Reply:
Mainly, the impact of this is that these types are any without the DOM types and then also show up in the globals if this package is imported. But maybe it doesn't matter given this has so few downloads (relatively speaking)